apps/home/templates/apps_home/home_page.html

```diff
@@ -3,7 +3,7 @@
 
 {% block content %}
 
-{% image page.header_image max-1920x560 as header_image %}
+{% image page.header_image original as header_image %}
```

Comment:
I got this from Opin, think it's fine as max pic size is 10mb so load time should be slow and we don't show on mobile anyway